src/FSharpPlus/FSharpPlus.fsproj

```diff
@@ -19,16 +19,15 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <Configurations>Debug;Release;Fable;Fable3;Test</Configurations>
+    <Configurations>Debug;Release;Fable;Test</Configurations>
     <Platforms>AnyCPU</Platforms>
     <LangVersion>8.0</LangVersion>
-    <LangVersion Condition=" '$(Configuration)' == 'Fable' OR '$(Configuration)' == 'Fable3' ">6.0</LangVersion>
+    <LangVersion Condition=" '$(Configuration)' == 'Fable' ">6.0</LangVersion>
```

Comment:
What should the language level for Fable4 be?

Reply:
I think we decided to drop support for Fable 3, but let's do it in another PR.

Reply:
I already did drop it in this PR. Note the removal of a lot of ifdefs...

Reply:
We can cherry-pick those commits out of this PR though.

Reply:
No, don't worry too much on that.